src/components/studies/adherenceHooks.ts

```diff
@@ -88,6 +88,43 @@ export const useAdherenceAlerts = (
   )
 }
 
+export const useUpdateAdherenceAlerts = () => {
+  const {token} = useUserSessionDataState()
+  const queryClient = useQueryClient()
+
+  // update alerts status on the server
+  const update = async (props: {
+    studyId: string, 
+    alertIds: string[], 
+    action: 'READ' | 'UNREAD' | 'DELETE'
+  }): Promise<any> => {
+    const {studyId, alertIds} = props
+    switch(props.action) {
```

Comment:
since we are going to have a single function in the service -- you don't need this switch - just pass the action param to the service function